share/completions/exif.fish

```diff
@@ -0,0 +1,76 @@
+function __get_target_files_tags
```

Comment:
This lives in the global namespace - we normally use a name like `__fish_exif_target_file_tags`